astroquery/nasa_exoplanet_archive/tests/test_nasa_exoplanet_archive.py

```diff
@@ -0,0 +1,271 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import print_function
+
+import sys
+import json
+import os
+from urllib.parse import urlencode
+
+import astropy.units as u
+import numpy as np
+import pkg_resources
+import pytest
+import requests
+from astropy.coordinates import SkyCoord
+from astropy.tests.helper import assert_quantity_allclose
+
+from ...utils.testing_tools import MockResponse
+from ..core import (
+    AstropyDeprecationWarning,
```

Comment:
this can be OK, but so far the preference was to import everything from their canonical location rather than from places where they are imported to but not part of the API

Reply:
Fixed!